pkg/apis/servicecatalog/types.go

```diff
@@ -266,10 +266,16 @@ type InstanceSpec struct {
 	// provisioned from.
 	PlanName string
 
-	// Parameters is a YAML representation of the properties to be
+	// Parameters is set of properties to be
 	// passed to the underlying broker.
 	Parameters *runtime.RawExtension
 
+	// Optional: AlphaSecretParameters is reference to the secret object containing
+	// sensitive information to pass to the underlying broker. This may be
+	// empty if no secret object is specified.
+	// +optional
+	AlphaSecretParameters *v1.LocalObjectReference
```

Comment:
@duglin Paul has published these review comments being unaware of the latest discussions in https://github.com/kubernetes-incubator/service-catalog/issues/621, you can ignore this thread.

I think we agreed in F2F follow-up discussions with Paul that pull request in its current state is working solution, but we need first to make sure we are allowed to use Secrets there (hence the need to sync up with Clayton and SIG Auth again, see Paul's comments below).

> rather than just using a single secret to hold everything

I would prefer to keep using a single Secret to keep things simple, generic and flexible.